index.html

```diff
@@ -1082,23 +1061,22 @@
         </section>
         <section>
           <h4>
-            Starting a presentation
+            Selecting a presentation display
           </h4>
           <p>
             When the <code><dfn for="PresentationRequest">start</dfn></code>
             method is called, the <a>user agent</a> MUST run the following
-            steps to <dfn>start a presentation</dfn>:
+            steps to <dfn>select a presentation display</dfn>, and then
+            <a>start a presentation connection</a> with the selected display.
```

Comment:
Although I don't see anyone could mis-interpret this and run the second algorithm twice, I would drop the end of the sentence.

In practice, the user agent will not run "select a presentation display", _and then_ "start a presentation connection". It will rather run "select a presentation display" which _includes_ a call to "start a presentation connection".